app/controllers/ctc/questions/stimulus_one_controller.rb

```diff
@@ -5,9 +5,14 @@ class StimulusOneController < QuestionsController
 
       layout "intake"
 
+      def self.show?(intake)
+        return false if intake.eip1_entry_method_calculated_amount?
+        true
+      end
+
       def update
-        current_intake.update!(eip1_amount_received: 0)
-        redirect_to questions_stimulus_two_path
+        current_intake.update!(eip1_amount_received: 0, eip1_entry_method: :did_not_receive)
```

Comment:
I think doing update right in the controller definitely simplifies the implementation, but we miss out on the ability to leverage the automatic reporting of the answers to mixpanel by doing this. I'm not sure if the original implementor knew that we make that tradeoff when we override update.

Then again, since this will be in the db it call also be looked at in mixpanel.

Reply:
This actually seems like the only page that doesn't use the form pattern... I'm wondering if its worth just converting it to use a form and default behavior of update.